lib/msal-common/src/server/RequestParameterBuilder.ts

```diff
@@ -0,0 +1,210 @@
+/*
+* Copyright (c) Microsoft Corporation. All rights reserved.
+* Licensed under the MIT License.
+*/
+
+import { AADServerParamKeys, SSOTypes } from "../utils/Constants";
+import { Constants } from "../utils/Constants";
+import { ScopeSet } from "../request/ScopeSet";
+import { ClientConfigurationError } from "../error/ClientConfigurationError";
+
+export class RequestParameterBuilder {
```

Comment:
This class as it is doesn't add any value. I think you should abstract the entire construction of the request into this class and get rid of the helpers that are here. See the above comment in `AuthorizationCodeClient`.